### PR TITLE
add rd.xml for System.Runtime.WindowsRuntime.UI.Xaml

### PR DESCRIPTION
**Pull request by luqunl**
System.Private.Interop need to create these 4 types through reflection, see https://github.com/dotnet/corert/blob/b18feac9f99f6d67c4e1182e5d2797e5529c3592/src/System.Private.Interop/src/WinRT/ExceptionHelpers.cs#L581

In AOT scenario, it is necessary to mark these types as dynamic to do reflection call.

### CODE REVIEW

File: src/System.Runtime.WindowsRuntime.UI.Xaml/src/Resources/System.Runtime.WindowsRuntime.UI.Xaml.rd.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Directives xmlns="http://schemas.microsoft.com/netfx/2013/01/metadata">
+  <Library Name="System.Runtime.WindowsRuntime.UI.Xaml">
+    <!-- System.Private.Interop will create the following type through reflection, see GetMappingExceptionForHR for more info-->
+    <Type Name="Windows.UI.Xaml.Markup.XamlParseException" Dynamic="Required All" />
```
Comment (morganbr) [inline src/System.Runtime.WindowsRuntime.UI.Xaml/src/Resources/System.Runtime.WindowsRuntime.UI.Xaml.rd.xml]: Can you narrow these to Activate="Required Public"? We shouldn't need the rest of these members to be reflectable.

**Comment by danmosemsft** [comment]
Serial port issues are https://github.com/dotnet/corefx/issues/20764 and can be ignored

**Comment by tijoytom** [comment]
test nnerloop netfx Windows_NT Release x64 Build and Test

**Comment by tijoytom** [comment]
@dotnet-bot help

**Comment by tijoytom** [comment]
@dotnet-bot test Innerloop OSX10.12 Release x64 Build and Test

**Pull request merged**